src/scss/_objects.stack.scss

```diff
@@ -0,0 +1,62 @@
+.grommet .stack {
+
+  &__label {
+    letter-spacing: normal;
+    margin-bottom: 0.25em;
+  }
+
+  &__headline {
+    margin-bottom: 0.125em;
+  }
+
+  &__paragraph {
+    &.grommetux-paragraph {
+      margin-top: 0.125em;
+      margin-bottom: 0.5em;
+    }
+  }
+
+  &__print {
+    &.grommetux-paragraph {
+      margin-bottom: 0.333em;
+    }
+  }
+
+  &__link {
+    margin-top: 0.75em;
+  }
+
+  &--small {
+    .stack__headline {
+      @include media-query(portable) {
+        font-size: 18px;
```

Comment:
Are there any Grommet variables that could be used for these font-sizes?  When this is pulled into Grommet, they will definitely question the use of custom px instead of the variables they've already defined.